components/component-stylesheets/badges.css

```diff
@@ -0,0 +1,60 @@
+.badge-icon{
+    font-size: 2.5rem;
+}
+
+.badge-icon-container{
+    position: relative;
+}
+
+.badge-img{
+    border-radius: 50%;
+    width: 8rem;
+    height: 8rem;
+}
+
+.status{
+    border-radius: 50%;
+    background-color: var(--mr-primary-color);
+    height: 1.5rem;
+    width: 1.5rem;
+    position : absolute;
+    border: 0.3rem solid white;
```

Comment:
You can use this white as a variable.

Reply:
thanks for pointing it out. fixed in recent commit.